.travis.yml

```diff
@@ -41,7 +41,7 @@ env:
         # are in astropy-ci-extras. If your package uses either of these,
         # add the channels to CONDA_CHANNELS along with any other channels
         # you want to use.
-        # - CONDA_CHANNELS='astopy-ci-extras astropy'
+        - CONDA_CHANNELS='astropy-ci-extras astropy'
```

Comment:
@pllim - Damn. My defence is that I was copy-pasting it from @bmorris3 (who probably copy pasted it from the original line, that I suppose I wrote...)